src/AppInstallerCLIE2ETests/GroupPolicy.cs

```diff
@@ -29,7 +29,7 @@ public void TearDown()
         public void PolicyEnableWinget()
         {
             GroupPolicyHelper.EnableWinget.Disable();
-            var result = TestCommon.RunAICLICommand("search", string.Empty);
+            var result = TestCommon.RunAICLICommand("search package", string.Empty);

```

Comment:
nit: this would be
var result = TestCommon.RunAICLICommand("search", "package");
Semantically the parameters are (Command, Args)